### DISCUSSION TITLE
Want to hide face and iris scan from biometric dialog (some Android10 and 11 devices)

### DISCUSSION THREAD

**Discussion by iamitksharma** [NiklasMerz/cordova-plugin-fingerprint-aio#338] [Q&A]
Want to hide face and iris scan from biometric dialog. Same to implement in native, I tried

import androidx.biometric.BiometricManager.Authenticators.BIOMETRIC_STRONG

and for BiometricPrompt:
val promptInfo = BiometricPrompt.PromptInfo.Builder()

                // 2
                .setTitle(getString(R.string.auth_title))
                // 3
                .setSubtitle(getString(R.string.auth_subtitle))
                // 4
                .setDescription(getString(R.string.auth_description))
                // 5
                .setNegativeButtonText("Cancel")
                **.setAllowedAuthenticators(BIOMETRIC_STRONG)**
                // 6
                .build()

But same tried to implement fingureprint-aio repository but not able to succeed. Can anyone help me?

**Comment by NiklasMerz** [comment]
This could probably a feature request if someone wants to work on this.
